.github/workflows/Kogito-Operator-PR-Check.yaml

```diff
@@ -0,0 +1,110 @@
+name: Kogito Operator PR Checks
+on:
+  pull_request:
+  push:
+    paths-ignore:
+      - "README.md"
+      - "examples/**"
+      - "LICENSE"
+      - "Makefile"
+    branches:
+      - master
+env:
+  OPERATOR_SDK_VERSION: v0.18.2
+  GO_VERSION: 1.14
+jobs:
+  golint:
+    name: Lint
+    runs-on: ubuntu-latest
+    env:
+      GOLANG_LINT_VERSION: v1.27.0
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Cache Operator SDK Binary
+        uses: actions/cache@v1
+        with:
+          path: /usr/bin/operator-sdk
```

Comment:
We can cache the entire `bin` directory instead in one step. This is something I need to change in the Nexus Operator as well.

EDIT: this will come in handy when we install openapi and KIND in this same dir.

Reply:
Okay, I'll cache the entire bin. @ricardozanini  Also, can you check if the actions are enabled in the repository settings?